Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -181,64 +183,60 @@ void GCPad::LoadDefaults(const ControllerInterface& ciface)
   EmulatedController::LoadDefaults(ciface);
 
   // Buttons
-  m_buttons->SetControlExpression(0, "X");  // A
-  m_buttons->SetControlExpression(1, "Z");  // B
-  m_buttons->SetControlExpression(2, "C");  // X
-  m_buttons->SetControlExpression(3, "S");  // Y
-  m_buttons->SetControlExpression(4, "D");  // Z
+  m_buttons->SetControlExpression(0, "`X`");  // A
+  m_buttons->SetControlExpression(1, "`Z`");  // B
+  m_buttons->SetControlExpression(2, "`C`");  // X
+  m_buttons->SetControlExpression(3, "`S`");  // Y
+  m_buttons->SetControlExpression(4, "`D`");  // Z
 #ifdef _WIN32
-  m_buttons->SetControlExpression(5, "RETURN");  // Start
+  m_buttons->SetControlExpression(5, "`RETURN`");  // Start
 #else
   // OS X/Linux
   // Start
-  m_buttons->SetControlExpression(5, "Return");
+  m_buttons->SetControlExpression(5, "`Return`");
 #endif
 
-  // stick modifiers to 50 %
-  m_main_stick->controls[4]->control_ref->range = 0.5f;
-  m_c_stick->controls[4]->control_ref->range = 0.5f;
```

Comment:
Did you intentionally remove those lines?

Reply:
No this is a mistake that happened when splitting up changes. thanks.